### PR TITLE
BD-2708: Condense "Managing custom data" and "Custom event and attribute management" articles 

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-2708**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md [hunk 1]
```diff
@@ -8,38 +8,84 @@ description: "This reference article covers how to manage custom data, such as p
...
 
-Blocklisting prevents a particular custom data object from being recorded by Braze going forward. Blocklisting does not remove data from user profiles or retroactively decrease the amount of data points incurred for that custom data object. 
+Blocklisting prevents a particular custom data object from being recorded by Braze going forward. However, blocklisting doesn't remove data from user profiles or retroactively decrease the amount of data points incurred for that custom data object. 
+
+For instructions and details about blocklisting custom data, refer to [Managing custom data]({{site.baseurl}}/user_guide/data_and_analytics/custom_data/managing_custom_data/#blocklisting-custom-data).
+
+### Blocklisting custom attributes, custom events, and products
+
+{% alert important %}
+Once an event or attribute is blocklisted, any segment, campaign, or Canvas using that event or attribute will be archived.
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md:59]:
```suggestion
When an event or attribute is blocklisted, any segment, campaign, or Canvas using that event or attribute will be archived.
```

File: _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md [hunk 2]
```diff
@@ -52,8 +98,40 @@ Deleting custom data does the following:
...
+|------------------|-------------|
+| Boolean | Inputs of `1`, `true`, `t` (not case sensitive) will be stored as `true` |
+| Boolean | Inputs of `0`, `false`, `f` (not case sensitive) will be stored as `false` |
+| Number | Integers or Floats (such as `1`, `1.5`) will be stored as numbers |
+{: .reset-td-br-1 .reset-td-br-2}
+
+For more information on specific filter options exposed by different data type comparisons, check out [Configuring reporting][43]. And for more information on the different available data types, refer to [Custom attribute data types][44].
+
+{% alert note %}
+Data sent to Braze is immutable and cannot be deleted or modified once we've received it. However, you can use any of the steps listed in the preceding sections to exercise control over what you're tracking in your dashboard.
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md:125]:
```suggestion
Data sent to Braze is immutable and cannot be deleted or modified after we've received it. However, you can use any of the steps listed in the preceding sections to exercise control over what you're tracking in your dashboard.
```

File: _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md [hunk 3]
```diff
@@ -8,38 +8,84 @@ description: "This reference article covers how to manage custom data, such as p
...
+{: start="3"}
+3. Click **Save**.
 
 ### Naming custom events and custom attributes
 
 Custom events and custom attributes are case-sensitive. Keep this in mind when your development team integrates these custom events or attributes later. They must name the custom events or attributes exactly as you named them here, or Braze will generate a different custom event or attribute.
 
+## Managing properties
+
+Once you have created a custom event or product, you can click **Manage Properties** for that event or product to add new properties, blocklist existing properties, and view which campaigns or Canvases use this property in a [trigger event]({{site.baseurl}}/user_guide/engagement_tools/campaigns/scheduling_and_organizing/delivery_types/triggered_delivery/#step-1-select-a-trigger-event).
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md:38]:
```suggestion
After you have created a custom event or product, you can click **Manage Properties** for that event or product to add new properties, blocklist existing properties, and view which campaigns or Canvases use this property in a [trigger event]({{site.baseurl}}/user_guide/engagement_tools/campaigns/scheduling_and_organizing/delivery_types/triggered_delivery/#step-1-select-a-trigger-event).
```

File: _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md [hunk 4]
```diff
@@ -8,38 +8,84 @@ description: "This reference article covers how to manage custom data, such as p
...
 {% alert note %}
 If you are using the [older navigation]({{site.baseurl}}/navigation), you can find these pages under **Manage Settings**.
 {% endalert %}
 
-![Navigate to Custom Attributes or Custom Events][21]{: style="max-width:90%;" }
+![Navigate to Custom Attributes or Custom Events or Products.][21]{: style="max-width:90%;" }
 
 {: start="2"}
-2. Add a new custom attribute or custom event.<br><br> For custom attributes, select a [data type][20] for this attribute (for instance, boolean or string). An attribute's data type will determine the segmentation filters available for that attribute. <br><br>![Add new attribute or event][22]{: style="max-width:70%;" }
+2. To add a custom attribute, event, or product, go to the respective page and click **Add Custom Attributes** or **Add Custom Events** or **Add Products**.<br><br> For custom attributes, select a [data type][20] for this attribute (for instance, boolean or string). An attribute's data type will determine the segmentation filters available for that attribute. <br><br>![Add new attribute or event][22]{: style="max-width:80%;" }
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md:28]: This is displaying as "3" in the deployment preview. 🤔 

https://braze-docs-5vs597owk-braze.vercel.app/docs/user_guide/data_and_analytics/custom_data/managing_custom_data/#pre-populating-custom-data

File: _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md [hunk 5]
```diff
@@ -8,38 +8,84 @@ description: "This reference article covers how to manage custom data, such as p
...
+{: start="3"}
+3. Click **Save**.
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md:30]: This displays as "1" in the deployment

**Review by rachel-feinberg — APPROVED** [review]
Left a few suggestions! Overall looks good!

File: _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md (hunk 4 above)
Comment (lydia-xie) [reply _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md]: Fixed it! Ty!

**Pull request merged**